github-orgs/kubeflow/org.yaml

```diff
@@ -211,6 +213,12 @@ orgs:
             - ziamsyed
             - sasha-gitg
             privacy: closed
+          IBM:
```

Comment:
We aren't sync'ing groups right now. The groups we have are manually curated and pretty much legacy.

I'd probably remove it for now. If you want to open an issue about supporting groups we can figure it out.

I'm not sure what we should do going forward.

Reply:
Thanks @jlewi , I removed this part in the PR, please help check.